symplyphysics/laws/waves/optical_path_length_from_geometric_path_length_and_refractive_index.py

```diff
@@ -0,0 +1,38 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless)
+
+# Description
+## The optical path length L between points A and B of a transparent medium is the distance over which light (optical radiation) has spread it would be in a vacuum for the same time it takes from A to B in the medium.
```

Comment:
You still write about paths between points A and B, while I'm convinced it is the same path. Check and fix your description.